Dockerfile

```diff
@@ -3,16 +3,18 @@ FROM ruby:2.6.2
 ADD Gemfile Gemfile
 ADD Gemfile.lock Gemfile.lock
 
+EXPOSE 3000
+
+ENV RAILS_LOG_TO_STDOUT true
+
 RUN bundle install
 
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && apt-get install -y nodejs
 
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
- && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
- && apt-get update -qq \
- && apt-get install -y yarn
+ADD . /verify-self-service/
+
+WORKDIR /verify-self-service
+
+CMD bundle exec puma -p 3000
```

Comment:
I think it might be simpler down the road when we start running elsewhere if we use the 'production' command here rather than the development command.

Reply:
yup, the `CMD` is for running locally. The actual command will be supplied from the Fargate task itself.